packages/api/es/queries.js

```diff
@@ -60,9 +59,6 @@ const build = {
       fieldName = match[1];
     }
 
-    // check for boolean values
```

Comment:
What is the associated bug that this is fixing?

Reply:
This is the actual bug. There were two issues (should have put that in the notes). First the dashboard was passing the level up as as a string and the API expects a number. I thought a string makes more sense. The other issue was the number that is getting passed in (it's indexed as a number) gets converted to a bool and then the query is invalid. I was trying to figure out why we'd ever want to convert the number to a bool by looking at the code and i couldn't find anything there.

Reply:
OK. Should at least we note this in the CHANGELOG so we remember that we made in the change in case it does turn out to be a problem?

Reply:
Yep, great point. I updated the Changelog.